system-addon/test/unit/lib/ManualMigration.test.js

```diff
@@ -38,17 +42,23 @@ describe("ManualMigration", () => {
       }
     };
 
-    const {ManualMigration} = injector({"lib/ActivityStreamPrefs.jsm": {Prefs: fakePrefs}});
+    // const {ManualMigration} = injector({"lib/ActivityStreamPrefs.jsm": {Prefs: fakePrefs}});
+    const {ManualMigration} = injector({"lib/ActivityStreamPrefs.jsm": {}});
```

Comment:
The `injector` was only needed to pass in a fake prefs, so because we're no longer using that, we should be able to just `import` ManualMigration as a regular module at the top of the test file.